src/form-builder.styles.tsx

```diff
@@ -31,3 +32,15 @@ export const ToastWrapper = styled.div`
     align-items: flex-end;
     gap: 1rem;
 `;
+
+export const ErrorWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
use `100%` so it can takes the parent's dimensions instead of the entire screen size because there may be other elements outside form builder - we dont want to cover those


```suggestion
    height: 100%;
```